core/tests/ui/desktop/e2e/checkout.spec.ts

```diff
@@ -70,9 +70,6 @@ test('Complete checkout as a logged in shopper', async ({ page }) => {
   await page.getByRole('heading', { level: 1, name: 'Your cart' }).click();
   await page.getByRole('button', { name: 'Proceed to checkout' }).click();
 
-  await page.getByLabel('Email').fill(faker.internet.email());
```

Comment:
This test was failing because with a logged in customer, we are able to fill the email on the checkout page and so, the test was just waiting to find and fill the email. 

As the test is already failing, I am fixing it as part of this same PR as the change is small.